airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceDatatypeTest.java

```diff
@@ -15,11 +16,25 @@
 import io.airbyte.integrations.util.HostPortResolver;
 import java.sql.SQLException;
 import org.jooq.SQLDialect;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.testcontainers.containers.PostgreSQLContainer;
 import org.testcontainers.utility.MountableFile;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
 
+@ExtendWith(SystemStubsExtension.class)
 public class PostgresSourceDatatypeTest extends AbstractPostgresSourceDatatypeTest {
 
+  @SystemStub
+  private EnvironmentVariables environmentVariables;
+
+  @BeforeEach
+  void setup() {
```

Comment:
We could just move this inside the `setupDatabase` method as first line. Dont need @BeforeEach method for it